ynput/OpenPype
# Max: fix the subset name not changing accordingly after the variant name changes

## Changelog Description
Resolve #5902

## Additional info
N/A
## Testing notes:
1. Create multiple instances in the same family with OP in Max
2. If you create instance with the same subset names, it will error out.
3. Go to Publish Tab
4. Changes the Variant Name, Asset Name and Task Name
5. If user changes the variant name of which already been used as subset name, it will raise the error too.
6. Make sure you have different variant names for subset, and publish. It is supposed that it won't have wrong error on publishing.
